v3/src/components/data-display/models/data-configuration-model.ts

```diff
@@ -358,27 +358,39 @@ export const DataConfigurationModel = types
       name: "valuesForAttrRole"
     })
   }))
+  .views(self => ({
+    numericValuesForAttribute(attrID: string) {
+      const dataset = self.dataset
+      const allCaseIDs = Array.from(self.visibleCaseIds)
+      const allValues = attrID
+        ? allCaseIDs.map((anID: string) => {
+          const value = dataDisplayGetNumericValue(dataset, anID, attrID)
+          return isFiniteNumber(value) ? value : null
+        }) : []
+      return allValues.filter(aValue => aValue != null)
+    },
```

Comment:
Just noting that this function does no caching, so if it is called repeatedly it could be expensive.